src/test/java/org/fcrepo/migration/f4clients/OCFLFedora4ClientTest.java

```diff
@@ -0,0 +1,48 @@
+/**
+ *
+ */
+package org.fcrepo.migration.f4clients;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+
+/**
+ * @author Remigiusz Malessa
+ * @since 4.4.1-SNAPSHOT
+ */
+public class OCFLFedora4ClientTest {
```

Comment:
Unit tests are helpful, particular in the context of test-driven-development. In order to ensure functionality does not break over time, integration-tests guard against regressions.
Could you also include an integration-test, along the lines of:
https://github.com/fcrepo4-exts/migration-utils/blob/master/src/test/java/org/fcrepo/migration/f4clients/OCFLGoLangFedora4ClientIT.java